README.md

```diff
@@ -86,63 +87,108 @@ void main() async {
 
   runApp(MyApp());
 }
-
-class MyApp extends StatefulWidget {
-  @override
-  MyAppState createState() => MyAppState();
-}
-
-class MyAppState extends State<MyApp> {
-  @override
-  void initState() {
-    super.initState();
-    // Load ads.
-  }
-}
 ```
 
-## Initialize the admob
+### Initialize the admob plugin
 
 ```dart
-import 'package:easy_ads_flutter/easy_ads_flutter.dart';
-
 EasyAds.instance.initAdmob(
         interstitialAdUnitId: InterstitialAd.testAdUnitId,
         rewardedAdUnitId: RewardedAd.testAdUnitId);
 ```
 
-## load ad
+### Initialize the unity plugin
 
 ```dart
-import 'package:easy_ads_flutter/easy_ads_flutter.dart';
+EasyAds.instance.initUnity(
+        unityGameId: 'unityGameId',
+        testMode: true,
+        interstitialPlacementId: 'Interstitial_Android',
+        rewardedPlacementId: 'Rewarded_Android');
+```
 
+### loading an ad
+
+```dart
 EasyAds.instance.loadRewardedAd();
 
 OR
 
 EasyAds.instance.loadInterstitialAd();
 ```
 
-## showing ad
+### showing Interstitial ad
 
 ```dart
-import 'package:easy_ads_flutter/easy_ads_flutter.dart';
-
 EasyAds.instance.showInterstitialAd();
+```
 
-OR
+### showing Rewarded ad
 
+```dart
 EasyAds.instance.showRewardedAd();
 ```
 
-## dispose ad
+### disposing an ad
 
 ```dart
-import 'package:easy_ads_flutter/easy_ads_flutter.dart';
+EasyAds.instance.disposeAds();
+```
 
-EasyAds.instance.disposeInterstitialAd();
+## Banner Ads
 
-OR
+#### Declare Banner ad
+
+Declare banner ad instance variable and banner ad placement ids
+
+```dart
+class _CountryDetailScreenState extends State<CountryDetailScreen> {
```

Comment:
it is okay but i would rather use _ExampleScreenState

Reply:
Yes, better. When you update read me, you consider that